### PR TITLE
Makes ranks better :tm:

### PR DESCRIPTION
**Pull request by Kmc2000**
## About The Pull Request

People keep complaining about ranks, so here's my submission:
The chad BSG13 ranks.

## Changelog
:cl: Kmc2000 with Six
add: Added a new rank config for people like me whose entire military knowledge comes from sci fi shows from the early 2000s.
/:cl:

### CODE REVIEW

File: config/ranks/bsg13.txt [hunk 1]
```diff
@@ -0,0 +1,52 @@
+Captain=CMDR
```
Comment (Kmc2000) [inline config/ranks/bsg13.txt]: Yep, captain is a commander. Upset?

File: config/ranks/bsg13.txt [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Executive Officer=COL
+Head of Security=LTCOL
+Chief Engineer=LTCOL
+Research Director=LTCOL
+Chief Medical Officer=LTCOL
+Master At Arms=MAJOR
+Warden=MAJOR
```
Comment (francinum) [inline config/ranks/bsg13.txt]:
```suggestion
Master At Arms=MAJ
Warden=MAJ
```

File: config/ranks/bsg13.txt [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+Bridge Staff=LT
+
+Assistant=ENS
+
+Quartermaster=MAJOR
```
Comment (francinum) [inline config/ranks/bsg13.txt]:
```suggestion
Quartermaster=MAJ
```

**Comment by Vasily2013** [comment]
thank fuck that this at least uses our variable system for ranks.

File: config/ranks/bsg13.txt [hunk 4]
```diff
@@ -0,0 +1,52 @@
+Captain=CMDR
+Executive Officer=COL
+Head of Security=LTCOL
```
Comment (covertcorvid) [inline config/ranks/bsg13.txt]: The correct abbreviation is LTC

File: config/ranks/bsg13.txt [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+Medical Doctor=LT
+Paramedic=LT2
+Chemist=LT2
+Geneticist=LT2
+Virologist=LT2
+
+Scientist=LT
+Roboticist=LT
+
+Detective=LT1
```
Comment (covertcorvid) [inline config/ranks/bsg13.txt]: Is he intended to be literally the only LT1 on the ship?

File: config/ranks/bsg13.txt [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+Head of Security=LTCOL
+Chief Engineer=LTCOL
+Research Director=LTCOL
+Chief Medical Officer=LTCOL
+Master At Arms=MAJ
+Warden=MAJ
+
+Bridge Staff=LT
+
+Assistant=ENS
```
Comment (captainwiseguy) [inline config/ranks/bsg13.txt]: you sure you want to make assistants officers?

File: config/ranks/bsg13.txt [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+Scientist=LT
+Roboticist=LT
+
+Detective=LT1
+Security Officer=LT
+Brig Physician=LT2
+
+Flight Leader=CPT
+Air Traffic Controller=LT
```
Comment (Vasily2013) [inline config/ranks/bsg13.txt]: What class of LT? 1 or 2?

File: config/ranks/bsg13.txt [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+Paramedic=LT2
+Chemist=LT2
+Geneticist=LT2
+Virologist=LT2
+
+Scientist=LT
+Roboticist=LT
+
+Detective=LT1
+Security Officer=LT
```
Comment (Vasily2013) [inline config/ranks/bsg13.txt]: What class of LT?

File: config/ranks/bsg13.txt [hunk 9]
```diff
@@ -0,0 +1,52 @@
+Captain=CMDR
+Executive Officer=COL
+Head of Security=LTCOL
+Chief Engineer=LTCOL
+Research Director=LTCOL
+Chief Medical Officer=LTCOL
+Master At Arms=MAJ
+Warden=MAJ
+
+Bridge Staff=LT
```
Comment (Vasily2013) [inline config/ranks/bsg13.txt]: What class of LT?

File: config/ranks/bsg13.txt [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+Janitor=CRW
+
+Clown=CRW
+Mime=CRW
+Curator=CRW
+Lawyer=CRW
+
+Chaplain=CRW
+
+Station Engineer=LT
```
Comment (Vasily2013) [inline config/ranks/bsg13.txt]: What class of LT?

File: config/ranks/bsg13.txt [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+Atmospheric Technician=LT
```
Comment (Vasily2013) [inline config/ranks/bsg13.txt]: What class of LT?

File: config/ranks/bsg13.txt [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+
+Medical Doctor=LT
```
Comment (Vasily2013) [inline config/ranks/bsg13.txt]: What class of LT?

File: config/ranks/bsg13.txt [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+Paramedic=LT2
+Chemist=LT2
+Geneticist=LT2
+Virologist=LT2
+
+Scientist=LT
```
Comment (Vasily2013) [inline config/ranks/bsg13.txt]: What class of LT?

File: config/ranks/bsg13.txt [hunk 14]
```diff
@@ -0,0 +1,52 @@
...
+Roboticist=LT
```
Comment (Vasily2013) [inline config/ranks/bsg13.txt]: What class of LT?

File: config/ranks/bsg13.txt [hunk 15]
```diff
@@ -0,0 +1,52 @@
...
+Research Director=LTCOL
+Chief Medical Officer=LTCOL
+Master At Arms=MAJ
+Warden=MAJ
+
+Bridge Staff=LT
+
+Assistant=ENS
+
+Quartermaster=MAJ
```
Comment (Vasily2013) [inline config/ranks/bsg13.txt]: Is the QM meant to outrank the Flight Leader? Major is one step above Captain.

File: config/ranks/bsg13.txt [hunk 16]
```diff
@@ -0,0 +1,52 @@
+Captain=CMDR
+Executive Officer=COL
+Head of Security=LTCOL
+Chief Engineer=LTCOL
+Research Director=LTCOL
+Chief Medical Officer=LTCOL
+Master At Arms=MAJ
```
Comment (Vasily2013) [inline config/ranks/bsg13.txt]: is MAA meant to be same rank as QM?

File: config/ranks/bsg13.txt [hunk 17]
```diff
@@ -0,0 +1,52 @@
...
+Virologist=LT2
+
+Scientist=LT
+Roboticist=LT
+
+Detective=LT1
+Security Officer=LT
+Brig Physician=LT2
+
+Flight Leader=CPT
```
Comment (Vasily2013) [inline config/ranks/bsg13.txt]: Might be a bit confusing having a captain on a ship that isn't the captain. Suggest bump this to Major if you don't just refactor.

**Comment by francinum** [comment]
Please put rank lists on the forums at https://nsv.beestation13.com/phpBB3/viewforum.php?f=20

**Pull request closed without merging**